content/json-ld/latest/events/event-bookingavailability.md

```diff
@@ -0,0 +1,221 @@
+| ------------- | ---------------- | --------- |
+| Authorization | Bearer {token}   | true      |
+| X-Api-Key     | {apiKey}         | true      |
+| Content-Type  | application/json | false     |
+
+## Set status on top event level
+
+When set on top level, all subEvents of the event will inherit the bookingAvailability from the top event level.
+
+### HTTP request
+
+```
+PUT /events/{eventId}/bookingAvailability
+```
+
+Update the bookingAvailability of the top event.
+
+### Resource properties
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| eventId	| uuid | unique identifier for an event | d595414a-13e0-4dd2-b4bd-706598427351 |
+
+### Request body
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| bookingAvailability	| object |  |  |
+| type	| string | bookingAvailability of the event. possible vallues: `Available`, `Unavailable` | Unavailable |
+
+### Response
+
+* `204 No Content` : request successful
+* `400 Bad Request` : incorrect method, payload or URI
+* `401 Unauthorized` : expired JWT or the user behind the JWT does not have permission to perform this request
+
+### Example
+
+**Request**
+
+```
+PUT https://io-test.uitdatabank.be/events/03116768-1abc-405a-93d7-ba6ede52fe09/bookingAvailability
+Content-Type: application/json
+Authorization: Bearer {token}
+X-Api-Key: {apiKey}
+
+{
+   "type":"Unavailable",
+}
+```
+
+**Response**
+
+The following are example responses.
+
+```
+204 No Content
+```
+
+```
+400 Bad Request
+
+{
+    "validation_messages": {
+        "type": "Invalid type provided"
+    },
+    "title": "Invalid payload.",
+    "type": "about:blank"
+}
+```
+
+```
+401 Unauthorized
+
+Token claims validation failed. This most likely means the token is expired.
+```
+
+
+## Set bookingAvailability on subEvent level
+
+When set on subEvent level, the bookingAvailability on top event level will be derived from the bookingAvailability values on the subEvent level.
+
+When UiTdatabank calculates the bookingAvailability for the top event level, the following logic is being applied:
+
+| bookingAvailability subEvents | Derived bookingAvailability on top event level |
+| ------------- | ---------------- | 
+| All available | available | 
+| All unavailable | unavailable | 
+| Mixed | available | 
+
+
+### HTTP request
+
+```
+PATCH /events/{eventId}/subEvents
+```
+
+Update the bookingAvailability of one or more subEvents.
+
+### Resource properties
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| eventId	| uuid | unique identifier for an event | d595414a-13e0-4dd2-b4bd-706598427351 |
+
+### Request body
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| id	| integer | a zero-based number of the subEvent you want to set the status for | 2 (this corresponds with the 3th subEvent of your event) |
+| bookingAvailability	| object |  |  |
+| type	| string | bookingAvailability of the event. possible vallues: `Available`, `Unavailable` | Unavailable |
```

Comment:
Will it be clear that this is a property inside the `bookingAvailability` object? It's on the same "level" as the `id` and `bookingAvailability` in this table.